Tests/Tests/Tests.m

```diff
@@ -12,6 +12,18 @@ @interface Tests : XCTestCase
 
 @implementation Tests
 
+- (User *)insertUserrWithRemoteID:(NSNumber *)remoteID
+                        localID:(NSString *)localID
+                           name:(NSString *)name inContext:(NSManagedObjectContext *)context
```

Comment:
You you mind moving `inContext:(NSManagedObjectContext *)context` to a new line also?